docs/documentation.html

```diff
@@ -26,8 +26,8 @@
 	<div class="right">
 		<!--#include virtual="../includes/_docs_banner.htm" -->
     <h1>Documentation</h1>
-    <h3>Kafka 0.11.0 Documentation</h3>
-    Prior releases: <a href="/07/documentation.html">0.7.x</a>, <a href="/08/documentation.html">0.8.0</a>, <a href="/081/documentation.html">0.8.1.X</a>, <a href="/082/documentation.html">0.8.2.X</a>, <a href="/090/documentation.html">0.9.0.X</a>, <a href="/0100/documentation.html">0.10.0.X</a>, <a href="/0101/documentation.html">0.10.1.X</a>, <a href="/0102/documentation.html">0.10.2.X</a>.
+    <h3>Kafka {{fullDotVersion}} Documentation</h3>
```

Comment:
I tried this when updating `kafka-site` and it didn't seem to work. Did you test it? Also, seems like it should be `dotVersion`.

Reply:
@guozhangwang you need to do 3 things to use Handlebar variables in this file.

1. Wrap this html in script tags with a new unique id:

`<script id="documentation-template" type="text/x-handlebars-template">**HTML MARKUP GOES HERE**</script>`

2. Specify a spot to inject the compiled HTML at the bottom of the file:

`<div class="p-documentation"></div>`

3. Update the Handlebars script in _footer.htm to look for the new id you just added so it knows to compile it. You'll see an array called "templates" in that file. Just add "documumentation" to the list (assuming you use the id in the example above)

Reply:
@guozhangwang this isn't working because that markup isn't getting compiled with Handlebars. 

This file is a little tricky because it does server side includes of several files that each get compiled with Handlebars too. Unfortunately you can't nest files that are getting run through Handlebars.js. 

If you want to use variables like this in the H3 of this file you're going to need to break up this file into further subsections that each get compiled and run through Handlebars.js. To do this you'd need to do these 3 things for each new section.

Wrap this html in script tags with a new unique id:
<script id="newsectionname-template" type="text/x-handlebars-template">**HTML MARKUP GOES HERE**</script>

Specify a spot to inject the compiled HTML at the bottom of the file:
<div class="p-newsectionname"></div>

Update the Handlebars script in _footer.htm to look for the new id you just added so it knows to compile it. You'll see an array called "templates" in that file. Just add "newsectionname" to the list.

Reply:
I tried to break down the documentation file into another `header` section but realized there are a few more seo links that could break.

So given that we are moving ak files to marked files in the future, I'll still do the manual updates instead of relying on the templates.